packages/opentelemetry-automatic-tracer/README.md

```diff
@@ -0,0 +1,11 @@
+# `opentelemetry-automatic-tracer`
```

Comment:
Does this mean to be called automatic tracer? I think just `opentelemetry-tracer` is fine since the user would expect such a tracer to just work out of the box, so `automatic` is redundant. This is especially true since the base tracer is called `basic-tracer` and not `tracer`.

Reply:
I am fine with the `opentelemetry-tracer` name, done.